Source/ParameterEncoding.swift

```diff
@@ -159,7 +164,7 @@ public struct URLEncoding: ParameterEncoding {
 
         guard let parameters = parameters else { return urlRequest }
 
-        if let method = HTTPMethod(rawValue: urlRequest.httpMethod ?? "GET"), encodesParametersInURL(with: method) {
+        if let method = HTTPMethod(rawValue: urlRequest.httpMethod ?? "GET"), destination.encodesParametersInURL(for: method) {
```

Comment:
We now have an extension on `URLRequest` which should return the method, without having to initialize it.

Reply:
Good catch, addressed in 9ea95f6f.